intellij-ext/gradle.properties

```diff
@@ -8,7 +8,7 @@ pluginVersion = 3.2.0
 
 # See https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
 # for insight into build numbers and IntelliJ Platform versions.
-pluginSinceBuild = 213
+pluginSinceBuild = 222
```

Comment:
Why was this necessary?

Reply:
sorry was just testing something with builds, fixed